client/src/scene/SceneGameArena.ts

```diff
@@ -83,13 +86,36 @@ export class SceneGameArena extends Phaser.Scene {
                 }
             })
         );
+
+        this.add.bitmapText(
+            BOARD_PX - 13.75 * TILE_SIZE,
+            BOARD_PX - 11 * TILE_SIZE,
+            "brawl",
+            "current event"
+        );
+        this.spectatorDecision = this.add
+            .bitmapText(
+                BOARD_PX - 13.75 * TILE_SIZE,
+                (BOARD_PX - 11 * TILE_SIZE) * 45,
+                "brawl",
+                ""
+            )
+            .setVisible(false);
```

Comment:
I don't see this being set true anywhere and I don't think we need it... I thought we'd always have something showing with the default text being something like "None"